src/app/resources/rate-resources/resources-rate.component.ts

```diff
@@ -0,0 +1,131 @@
+import { switchMap } from 'rxjs/operators';
+import { Component, OnInit } from '@angular/core';
+import { CouchService } from '../../shared/couchdb.service';
+import { UserService } from '../../shared/user.service';
+
+import { ActivatedRoute, ParamMap } from '@angular/router';
+import { DomSanitizer } from '@angular/platform-browser';
+import { Router } from '@angular/router';
+import { environment } from '../../../environments/environment';
+
+import { MatFormField, MatFormFieldControl } from '@angular/material';
+import {
+    FormBuilder,
+    FormControl,
+    FormGroup,
+    FormArray,
+    Validators
+  } from '@angular/forms';
+
+@Component({
+  templateUrl: './resources-rate.component.html'
+})
+export class ResourcesRateComponent implements OnInit {
+
+  ratingForm: FormGroup;
```

Comment:
please follow tslint rule for indentation

Reply:
@razu9861 https://github.com/ole-vi/planet/pull/270/commits/06a0b4bef34cf7d1fb438d66817345ae3496b839